libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergContainerFragment.java

```diff
@@ -212,7 +212,11 @@ public void updateTheme(Bundle editorTheme) {
     }
 
     public void updateCapabilities(GutenbergPropsBuilder gutenbergPropsBuilder) {
-        GutenbergProps gutenbergProps = gutenbergPropsBuilder.build(getActivity(), mHtmlModeEnabled);
-        mWPAndroidGlueCode.updateCapabilities(gutenbergProps);
+        // We want to make sure that activity isn't null
+        // as it can make this crash to happen: https://github.com/wordpress-mobile/WordPress-Android/issues/13248
+        if (getActivity() != null) {
+            GutenbergProps gutenbergProps = gutenbergPropsBuilder.build(getActivity(), mHtmlModeEnabled);
```

Comment:
Perhaps extract `getActivity()` out to a variable to just insure that we never hit the scenario where the call on line 217 an object, but the second `getActivity()` call on line 218 to returns `null`.

```
final Activity activity = getActivity();
if (activity !=null) {
  GutenbergProps gutenbergProps = gutenbergPropsBuilder.build(activity, mHtmlModeEnabled);
  ...
}
```

Reply:
Ah good one. Fixed!